main/php.h

```diff
@@ -95,8 +95,6 @@
 typedef int uid_t;
 typedef int gid_t;
 typedef char * caddr_t;
-typedef unsigned int uint;
-typedef unsigned long ulong;
```

Comment:
@nikic, @petk, these two typedefs are [required by Xdebug](https://ci.appveyor.com/project/derickr/xdebug/builds/25269152/job/o023kswjv7psp0r8#L776) (and maybe other extensions) on Windows; should they really have been removed? Note that the are inside a `#ifdef PHP_WIN32`.

Reply:
In what they are they "required"? Can't they be replaced by the standard C types?

Reply:
Well, they are currently *used*; of course, they could be replaced by standard C types (or typedef'fed in Xdebug).

Reply:
Then I'd recommend replacing them... From a quick look, most of the uses are wrong anyway (uint should be uint32_t).

Reply:
Thanks @cmb69 for documenting the change, and @nikic for the info. +1 for using better practices in the extensions... Defining these back (via build system) and not using them in the php-src is from my point of view is not good practice. Specially in this case, for example. They should in worst case scenario be defined in the extension's config.m4 and windows headers file for particular extension instead (if that's possible).